mkdocs.yml

```diff
@@ -12,7 +12,9 @@ nav:
   - Resource monitoring dashboards: docs/app-monitoring/resource-monitoring-dashboards.md
   - Check application health after an outage: docs/app-monitoring/check-application-health-after-outage.md
   - User Defined Monitoring in OpenShift: docs/app-monitoring/user-defined-monitoring.md
-  - Best practices for application logging in OpenShift: docs/app-monitoring/best-pratices-for-application-logging-in-openshift.md
+  - Best practices for application logging in OpenShift: docs/app-monitoring/best-practices-for-application-logging-in-openshift.md
+  - SRE guidelines for platform shared services: docs/app-monitoring/sre-guidelines-for-platform-shared-services.md
+  - Guidelines for SLI and monitoring: docs/guidelines-for-sli-and-monitoring.md
```

Comment:
```suggestion
  - Guidelines for SLI and monitoring: docs/app-monitoring/guidelines-for-sli-and-monitoring.md
```

Reply:
Link wasn't working properly - needed to add the category first to be indexed in the left navigation. Apologies!